src/app/components/text-expand-repeater/index.html

```diff
@@ -22,6 +22,12 @@
     >
     Specifies the number of items to display before truncating the list. This property accepts <sky-code>number</sky-code> values.
     </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="itemTemplate"
+      isOptional="true"
+    >
+    Specifies the template for items in the list. This property accepts <sky-code>any</sky-code> values.
```

Comment:
This should be `TemplateRef<any>`:
https://github.com/blackbaud/skyux-layout/blob/master/src/app/public/modules/text-expand-repeater/text-expand-repeater.component.ts#L45

Reply:
Should we link to https://angular.io/api/core/TemplateRef, @Blackbaud-SteveBrush? i.e., "This property accepts `TemplateRef<any>` values." where "`TemplateRef<any>` values" is a link to that Angular documentation?

Reply:
Good idea! We're doing that for the inline-form `template` input as well.

Reply:
See: https://github.com/blackbaud/skyux2-docs/blob/master/src/app/components/inline-form/index.html#L23